### PR TITLE
Add core-js tests to increase coverage

### PR DESCRIPTION
**Pull request by mjlescano**

### CODE REVIEW

File: packages/core-js/test/utils/logger.test.js [hunk 1]
```diff
@@ -1,37 +1,100 @@
...
+    equal(logger.postpend, '');
+    equal(logger.boxing, false);
+  });
+
+  describe('when quiet is not enabled', () => {
+    before('disable quiet', () => {
+      logger.quiet = false;
+    });
+
+    it('does not record any logs', () => {
```
Comment (leomassazza) [inline packages/core-js/test/utils/logger.test.js:35]: in fact, it record logs :)

**Review by i-stam — DISMISSED** [review]
!!!

File: packages/core-js/test/utils/rpc.test.js [hunk 2]
```diff
@@ -21,7 +21,6 @@ describe('utils/rpc.js', () => {
     let snapshotId;
 
     before('call createSnapshot', async () => {
-      console.log(provider);
```
Comment (leomassazza) [inline packages/core-js/test/utils/rpc.test.js:24]: 🤦‍♂️ thanks 👍

File: packages/core-js/test/utils/logger.test.js (hunk 1 above)
Comment (i-stam) [reply packages/core-js/test/utils/logger.test.js:35]: Is it a copy-paste typo?

Comment (mjlescano) [reply packages/core-js/test/utils/logger.test.js]: ahah the copy-paster 🙈 (fixed)

**Review by leomassazza — APPROVED** [review]
LGTM

**Review by i-stam — APPROVED** [review]

**Pull request merged**